test/test_main.cpp

```diff
@@ -125,7 +125,10 @@ std::unique_ptr<PersistentStore> DatabaseContext::makePersistentStore() const{
 	                                                            clientConfig,
 	                                                            getPortalUserConfigPath(),
 	                                                            getEncryptionKeyPath(),
-	                                                            "",0, "slateci.net"));
+	                                                            "",
+																0,
```

Comment:
The preceding two lines have incorrect whitespace as not just the indent but also the alignment portion are formed from tabs.

Reply:
Hi @cnweaver , thanks for the feedback.  It's much appreciated.  I've updated the README. 

I've been running the unit tests using https://github.com/slateci/slate-client-server/blob/opentelemetry/resources/TESTING_SETUP.md .  The setup has probably drifted a bit from your setup.  I think  we've moved over to using clion for dev and as part of that we have a container ( https://github.com/slateci/slate-client-server/blob/opentelemetry/resources/docker/clion_remote.Dockerfile ) that sets things up for clion (vscode should be similar).  Hopefully this allows new developers to quickly get a build environment setup and get started. With this setup, builds happen in the container but you can debug using a local clion ide (i.e. set breakpoints in the clion ide, inspect variables) while clion runs and debugs the code inside the container.

Right now I have the unit tests use a provider to populate the signoz instance.  See the internal slate channel (or contact me directly) to get the address and login details if you're interested in the output.  This is just to get startup traces but I'll swap it out with a no-op in a further pr since there's a few other changes I need to make to span generation to make it display a bit better.

Reply:
Oh, I understand about the whitespace changes.  The tab vs space issue is a bit annoying especially since I thought I fixed it and I really don't see any alignment issues when editing code in clion.  I'll look into the formatting settings and trying to adjust that and some of the other stuff when I get a chance.

Reply:
It would be good not to break the ability to run tests without containers; a container system (and also some kind of IDE?) is a very hefty dependency, and closing off the ability to develop without containers is often a path to bitrot and loss of portability. Testing the setup to be deployed is great; being able to test in other contexts is valuable too. It seems like it should be simple to let the tests run in either type of environment.